CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Add `prop` function with `KProperty1` argument.
+
+### Deprecated
+- Deprecated `prop` function with `KCallable` argument.
```

Comment:
Nice job adding this! Would add the note about migrating to the version that takes an explicit name and lambda.

Reply:
Will do!